developers/verifiers/multicredentials.md

```diff
@@ -20,34 +20,70 @@ Here's how to get a demo Veres Wallet if you need one:
 
-Now that you have several VCs in your wallet, let's get them all verified in one request.
+**NOTE:** The credential JSON document now has an absolute URL for it's `id` value and a new `proof` section. The `issuer.id` has also been updated to reflect the playground as the Issuer, and (of course) the `issuanceDate` has also been set.
 
-1. Visit the playground's Verifier section at <https://playground.chapi.io/verifier>.
+That credential may now be stored in a wallet. Click "Store in Wallet" to proceed.
 
-2. Click each of the VCs you loaded into your wallet (i.e. "Alumni", "JFF x vc-edu PF3", and "Permanent Resident" for example).
+A modal will appear - similar to the one below - explaining that the playground wants to send the cerdential for you.
 
-3. Click "Request Verifiable Presentation" to request that your wallet share these credentials.
+![Playground offering to store a credential. The screen contains two buttons: one to cancel the request and one to go to the next screen.](playground-offering-to-store-credential.jpg)
 
-4. In the demo Veres Wallet you'll see each of the credentials you selected (plus the ability to select which ones you want to verify). Click the "Share" button to send them to the playground for verification.
+Clicking "Next" should present you with a list of available wallets:
 
-5. The process should end with a VerifiablePresentation JSON document and the words "Presentation Verified".
+![Choose a wallet modal presenting all preregisterd wallet systems which can be clicked on to proceed to storing the credentials there.](choose-a-wallet.jpg)
+
+However, if you instead receive a browser window popup warning you that you do not yet have one set up, you will need to return to the [Wallet Sign Up section](#wallet-sign-up) above.
+
+If you received a list of wallet options, pick the one you prefer by clicking on it's name to proceed.
+
+Each wallet will have it's own system for displaying the credential and offering to store it for you.
+
+Once you've clicked "Store" in your preferred wallet system, you will be brought back to the main playground screen.
+
+Follow the above process a few more times by clicking "Issuer Demo" to load your wallet with more example credentials.
+
+## Multiple Verifiable Credential Verification
+
+Now that you have several Verifiable Credentials (VCs) in your wallet, let's get them all verified in one request.
+
+First, visit the [playground's Verifier section](https://playground.chapi.io/verifier).
+
+Select the VCs you loaded into your wallet (i.e. "Alumni", "JFF x vc-edu PF3", and "Permanent Resident").
+
+![Select multiple credentials to submit them for verification.](select-multiple-credentials.jpg)
+
+Next, click "Request Verifiable Presentation" to request that your wallet share these credentials with the Playground's Verifier.
+
+In the demo Veres Wallet you'll see each of the credentials you selected (plus the ability to select which ones you want to verify).
```

Comment:
I think this is the first assumption of Veres. Maybe we should add a sentence saying "your wallet may display this presentation request differently"